packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -180,211 +203,227 @@ It took ${timeInMsToDownloadAll}ms to execute download() for all binaryTargets.`
+            "size": 23883344,
+          },
+          {
+            "name": "migration-engine-linux-musl",
+            "size": 19058352,
+          },
+          {
+            "name": "migration-engine-linux-musl-arm64-openssl-1.1.x",
+            "size": 22553584,
+          },
+          {
+            "name": "migration-engine-linux-musl-arm64-openssl-3.0.x",
+            "size": 24279432,
+          },
+          {
+            "name": "migration-engine-linux-musl-openssl-3.0.x",
+            "size": 19348096,
+          },
+          {
+            "name": "migration-engine-linux-static-arm64",
+            "size": 21218896,
+          },
+          {
+            "name": "migration-engine-linux-static-x64",
+            "size": 22324992,
+          },
+          {
+            "name": "migration-engine-rhel-openssl-1.0.x",
+            "size": 22144168,
+          },
+          {
+            "name": "migration-engine-rhel-openssl-1.1.x",
+            "size": 19440080,
+          },
+          {
+            "name": "migration-engine-rhel-openssl-3.0.x",
+            "size": 19441160,
+          },
+          {
+            "name": "migration-engine-windows.exe",
+            "size": 15780864,
+          },
+          {
+            "name": "query-engine-darwin",
+            "size": 17840936,
+          },
+          {
+            "name": "query-engine-darwin-arm64",
+            "size": 16655984,
+          },
+          {
+            "name": "query-engine-debian-openssl-1.0.x",
+            "size": 19287104,
+          },
+          {
+            "name": "query-engine-debian-openssl-1.1.x",
+            "size": 16808072,
+          },
+          {
+            "name": "query-engine-debian-openssl-3.0.x",
+            "size": 16808072,
+          },
+          {
+            "name": "query-engine-linux-arm64-openssl-1.0.x",
+            "size": 16722600,
+          },
+          {
+            "name": "query-engine-linux-arm64-openssl-1.1.x",
+            "size": 17462856,
+          },
+          {
+            "name": "query-engine-linux-arm64-openssl-3.0.x",
+            "size": 18788592,
+          },
+          {
+            "name": "query-engine-linux-musl",
+            "size": 16664640,
+          },
+          {
+            "name": "query-engine-linux-musl-arm64-openssl-1.1.x",
+            "size": 17708728,
+          },
+          {
+            "name": "query-engine-linux-musl-arm64-openssl-3.0.x",
+            "size": 19120392,
+          },
+          {
+            "name": "query-engine-linux-musl-openssl-3.0.x",
+            "size": 16775312,
+          },
+          {
+            "name": "query-engine-linux-static-arm64",
+            "size": 15984368,
+          },
+          {
+            "name": "query-engine-linux-static-x64",
+            "size": 19274912,
+          },
+          {
+            "name": "query-engine-rhel-openssl-1.0.x",
+            "size": 19287104,
+          },
```

Comment:
Wow, that is quite an increase. What happened there?

Reply:
https://github.com/prisma/prisma-engines/pull/3932.

Basically we've been statically linking more modern OpenSSL in 1.0.x engine variants since https://github.com/prisma/prisma-engines/pull/1770, but at some point it broke for QE binary and it has been using system OpenSSL while migration engine and QE library have been using their own one. Then the non-functional cargo feature was fixed, and we implemented https://github.com/prisma/ecosystem-tests/pull/3500 to prevent things like this happening without us knowing.

We discussed this on a meeting with you, Tom and Pierre a few months ago.

Reply:
Understood.